src/fetch_foursquare_info.py

```diff
@@ -32,98 +37,99 @@ def load_cnpjs(subquota_description):
-    relevant_cnpjs = load_cnpjs('Congressperson meal')
-    return all_companies[all_companies['clean_cnpj'].isin(relevant_cnpjs)]
+    all_companies['clean_cnpj'] = all_companies['cnpj'].map(only_numbers)
+    return all_companies[all_companies['clean_cnpj'].isin(cnpjs)]
 
-def cleanup_cnpj(cnpj):
-    return re.sub("\D", "", cnpj)
+def only_numbers(string):
+    """Return a string w only the numbers from the given string"""
+    return re.sub("\D", "", string)
 
-def remaining_companies(companies):
-    fetched_companies = load_foursquare_companies_dataset()
+def remaining_companies(companies, fetched_companies):
+    """Return the first DF but without matching CNPJs from the second DF"""
     return companies[~companies['cnpj'].isin(fetched_companies['cnpj'])]
 
 def load_foursquare_companies_dataset():
+    """Return a DF with the data already collected"""
     if os.path.exists(FOURSQUARE_DATASET_PATH):
-        return pd.read_json(FOURSQUARE_DATASET_PATH)
+        return pd.read_csv(FOURSQUARE_DATASET_PATH)
     else:
         return pd.DataFrame(columns=['cnpj'])
 
 def get_venue(company):
+    """Return a matching venue from Foursquare for the given company Series"""
     venue = search(company)
     if venue:
-        return fetch_venue_by_id(venue['id'])
+        return fetch_venue(venue['id'])
 
-def search(company, timeout=5):
+def search(company):
+    """Search Foursquare's API for a match for a given company Series"""
     params = dict(DEFAULT_PARAMS)
-    params.update({
-        'query': company['trade_name'],
-        'll': '%s,%s' % (company['latitude'], company['longitude']),
-        'zip_code': company['zip_code'],
-        'intent': 'match'
-    })
+    params.update({ 'query': company['trade_name'],
+                    'll': '%s,%s' % (company['latitude'], company['longitude']),
+                    'zip_code': company['zip_code'],
+                    'intent': 'match' })
     url = 'https://api.foursquare.com/v2/venues/search'
     response = requests.get(url, params=params)
-
-    return parse_search_results(response)
+    if 200 >= response.status_code < 300:
+        return parse_search_results(response)
 
 def parse_search_results(response):
+    """Return the first venue from the given search response"""
     json = response.json()
-    if json['meta']['code'] == 200:
-        venues = json['response']['venues']
-        if venues:
-            return venues[0]
+    venues = json['response']['venues']
+    if venues:
+        return venues[0]
 
-def fetch_venue_by_id(venue_id):
+def fetch_venue(venue_id):
+    """Return specific data from Foursquare for the given venue_id"""
     url = 'https://api.foursquare.com/v2/venues/%s' % venue_id
     response = requests.get(url, params=DEFAULT_PARAMS)
-    return parse_venue_info(response)
+    if 200 >= response.status_code < 300:
+        return parse_venue_info(response)
 
 def parse_venue_info(response):
```

Comment:
Taking into account what I just said about `parse_search_results(…)`, this could be simplified as:

```python
def parse_venue_info(response):
    """Return only venue data from the given fetch_venue response"""
    json_response = response.json()  # don't overwrite native json module
    return json_response.get('response', {}).get('venue')  # returns the venue info or None
```

Surely you can delete my comments ; )